packages/vite/src/node/ssr/ssrStacktrace.ts

```diff
@@ -71,3 +71,19 @@ export function rebindErrorStacktrace(e: Error, stacktrace: string): void {
     e.stack = stacktrace
   }
 }
+
+const rewroteStacktraceSymbol = Symbol('Vite.rewroteStacktrace')
+
+export function ssrFixStacktrace(
+  e: Error & { [rewroteStacktraceSymbol]?: true },
+  moduleGraph: ModuleGraph
+): void {
+  if (!e.stack) return
+  // stacktrace shouldn't be rewritten more than once
+  if (e[rewroteStacktraceSymbol]) return
```

Comment:
Maybe we could do this check with a WeakMap against the Error object? Or is this information useful outside of this function?

Reply:
Yeah, that looks better. I've updated!